easybuild/easyconfigs/s/Szip/Szip-2.1.1-foss-2017a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Szip'
+version = '2.1.1'
+
+homepage = 'http://www.hdfgroup.org/doc_resource/SZIP/'
+description = "Szip compression software, providing lossless compression of scientific data"
+
+toolchain = {'name': 'foss', 'version': '2017a'}
```

Comment:
using `GCCcore/6.3.0` here is better, to allow sharing this between `foss/2017a` and `intel/2017a`?